### PR TITLE
PODS-9931: Update to HTTPClientV2

### PR DESCRIPTION
**Pull request by nonebula**

### CODE REVIEW

File: app/handlers/ErrorHandlerWithReturnLinkToManage.scala [hunk 1]
```diff
@@ -17,25 +17,30 @@
...
+
 import javax.inject.{Inject, Singleton}
 import models.Link
 import play.api.i18n.{I18nSupport, Lang, MessagesApi}
-import play.api.mvc.Request
+import play.api.mvc.{Request, RequestHeader}
 import play.twirl.api.Html
 import views.html.{error_template, error_template_page_not_found}
 
+import scala.concurrent.ExecutionContext.Implicits.global
```
Comment (PavelVjalicin) [inline app/handlers/ErrorHandlerWithReturnLinkToManage.scala:28]: Introduce (implicit ec: ExecutionContext) as an injectable class inside of class constructor instead.

**Review by PavelVjalicin — CHANGES_REQUESTED** [review]

File: app/handlers/ErrorHandler.scala [hunk 2]
```diff
@@ -17,29 +17,43 @@
...
+                                    )(implicit request: RequestHeader): Future[Html] = {
+    def requestImplicit: Request[_] = Request(request, "")
+    def messages: Messages = messagesApi.preferred(request)
+    Future.successful(errorTemplateView(pageTitle, heading, message, appConfig)(requestImplicit, messages))
   }
 
-  override def standardErrorTemplate(pageTitle: String, heading: String, message: String)
-                                    (implicit rh: Request[_]): Html =
-    errorView(pageTitle, heading, message, appConfig)
+//  def standardErrorTemplate(pageTitle: String, heading: String, message: String)(implicit request: RequestHeader): Future[Html] = {
```
Comment (jamdade-sharad) [inline app/handlers/ErrorHandler.scala:51]: remove commented code

Comment (nonebula) [reply app/handlers/ErrorHandler.scala]: Done.

**Review by jamdade-sharad — APPROVED** [review]
approved

**Pull request merged**